dev/python_release.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e -pipe
+
+# Switch to the project root directory
+cd $( dirname $0 )
+cd ..
+
+# Clean up uncommitted files
+#git clean -fdx
+
+cd python
+cd delta-kernel-rust-sharing-wrapper
+rm -Rf .venv
+python3 -m venv .venv
+source .venv/bin/activate
+pip install maturin
+maturin build
+
```

Comment:
L10 - L17 seems not needed if we are using github actions to release the sharing wrapper.

Reply:
yeah ill delete this